Empire/Empire.cpp

```diff
@@ -1118,7 +1118,7 @@ void ProductionQueue::Update() {
             std::tie(item_cost, build_turns) = queue_item_costs_and_times[key];
             float total_item_cost = item_cost * element.blocksize;
 
-            float additional_pp_to_complete_element = total_item_cost * (1.0f - element.progress); // additional PP, beyond already-accumulated PP to finish this element once
+            float additional_pp_to_complete_element = total_item_cost * (element.remaining - element.progress); // additional PP, beyond already-accumulated PP to finish this element once
```

Comment:
I just noticed that the comment at the end of that line is also now mismatched to the calculation.  Howabout I just revert the whole line to its status prior to c6cc24013:
```
float additional_pp_to_complete_element = total_item_cost * element.remaining - element.progress*total_item_cost; // additional PP, beyond already-accumulated PP, to build all items in this element
```

Since you had changed both the calculation and the comment in that commit, it looks like perhaps you had been considering changing the way that additional_pp_to_complete_element would be put to use, but the change slipped into the grooming commit before you made the followup change to how the variable was actually being used.

One thing to keep in mind: I've been working on reorganizing this whole method, in prep for the Imperial Stockpile, and I've concluded that in this whole subsection section about figuring out the smallest possible max_turns for the following loop, and using additional_pp_to_complete_element to maybe avoid the loop altogether, I really just added a fair bit of code complexity for practically no real benefit.  Most of this whole subsection is eliminated in a cleanup PR that I will probably be submitting later today, but I wanted to get in a simple fix for this bug as a standalone commit so that it would be most suitable to include in any bugfix release we might wind up doing for 0.4.7.